migrator/bin/ups-migrator

```diff
@@ -0,0 +1,175 @@
+#!/bin/sh
```

Comment:
not sure, but I think this could be moved to something like `src/main/shell`, or is this meant to be an executable by itself ?